js/components/oppfolgingsdialog/godkjennplan/releasetplan/GodkjentPlan.js

```diff
@@ -37,6 +38,14 @@ const textBothApprovedOppfolgingsplan = (arbeidstakerName) => {
     return `Denne versjonen av planen er godkjent av ${arbeidstakerName} og deg.`;
 };
 
+const textForcedApprovedOppfolgingsplan = (rootUrl, arbeidstakerName) => {
+    return (<BildeTekstLinje
+        imgUrl={`${rootUrl}/img/svg/report-problem-circle.svg`}
+        alt="info"
+        tekst={`Du har ferdigstilt planen uten godkjenning fra ${arbeidstakerName}.`}
```

Comment:
Vi har pleid  å tekster og interpolerte tekster i egne objekter/funksjoner for å abstrahere ut tekst fra selv representasjonslogikk. 
Et eksempel i denne filen er:
`const textBothApprovedOppfolgingsplan = (arbeidstakerName) => {
    return `Denne versjonen av planen er godkjent av ${arbeidstakerName} og deg.`;
};`
og 
`const texts = {
    godkjentPlan: {
        title: 'Oppfølgingsplanen',
    },
    godkjentPlanUtvidbar: {
        title: 'Se planen',
        requestFeilet: 'Beklager, vi kunne ikke hente dokumentet på dette tidspunktet. Prøv igjen senere!',
    },
    avbrytPlanBekreftelse: {
        title: 'Ønsker du å endre planen?',
        info: 'Hvis du endrer planen, må du sende den til godkjenning hos den andre. Etter godkjenning blir den en gjeldende plan.',
        button: 'Gjør endringer',
    },
};`